runtime/lang/menu_ru_ru.utf-8.vim

```diff
@@ -1,337 +1,1266 @@
+menutrans Toggle\ Relati&ve\ Line\ Numbering<Tab>:set\ rnu! Относите&льная\ нумерация\ строк<Tab>:set\ nru!
+menutrans Toggle\ &List\ Mode<Tab>:set\ list! Отобра&жение\ невидимых\ символов<Tab>:set\ list!
+menutrans Toggle\ Line\ &Wrapping<Tab>:set\ wrap! &Перенос\ длинных\ строк<Tab>:set\ wrap!
+menutrans Toggle\ W&rapping\ at\ Word<Tab>:set\ lbr! Перенос\ &целых\ слов<Tab>:set\ lbr!
+" menutrans Toggle\ Tab\ &Expanding<Tab>:set\ et! TRANSLATION\ MISSING
+menutrans Toggle\ &Auto\ Indenting<Tab>:set\ ai! Автоматическое\ форматирование\ &отступов<Tab>:set\ ai!
+menutrans Toggle\ &C-Style\ Indenting<Tab>:set\ cin! Форматирование\ отступов\ в\ &стиле\ C<Tab>:set\ cin!
+
+" other options
+menutrans &Shiftwidth Вели&чина\ отступа
+menutrans Soft\ &Tabstop Ширина\ &табуляции
+menutrans Te&xt\ Width\.\.\. &Ширина\ текста\.\.\.
+menutrans &File\ Format\.\.\. &Формат\ файла\.\.\.
+
+" fun! s:TextWidth()
+if !exists("g:menutrans_textwidth_dialog")
+  let g:menutrans_textwidth_dialog = "Введите ширину текста для форматирования.\nДля отмены форматирования введите 0."
+endif
+
+" fun! s:FileFormat()
+if !exists("g:menutrans_fileformat_dialog")
+  let g:menutrans_fileformat_dialog = "Выберите формат файла."
+endif
+if !exists("g:menutrans_fileformat_choices")
+  let g:menutrans_fileformat_choices = "&Unix\n&Dos\n&Mac\nО&тмена"
+endif
+" }}}
+" menutrans Show\ C&olor\ Schemes\ in\ Menu TRANSLATION\ MISSING
+menutrans C&olor\ Scheme &Цветовая\ схема
+" menutrans None TRANSLATION\ MISSING
+" menutrans Show\ &Keymaps\ in\ Menu TRANSLATION\ MISSING
+menutrans &Keymap Раскладка\ кл&авиатуры
+menutrans Select\ Fo&nt\.\.\. Выбор\ &шрифта\.\.\.
+" }}}
+
+" Programming menu
+menutrans &Tools &Инструменты
+" Tools menuitems {{{1
+menutrans &Jump\ to\ This\ Tag<Tab>g^] &Перейти\ к\ метке<Tab>g^]
+menutrans Jump\ &Back<Tab>^T &Вернуться\ назад<Tab>^T
+menutrans Build\ &Tags\ File Создать\ файл\ ме&ток
+
+" Tools.Spelling Menu
+menutrans &Spelling Пр&авописание
+" Tools.Spelling menuitems and dialog {{{2
+menutrans &Spell\ Check\ On &Вкл\ проверку\ правописания
+menutrans Spell\ Check\ &Off Вы&кл\ проверку\ правописания
+menutrans To\ &Next\ Error<Tab>]s &Следующая\ ошибка<Tab>]s
+menutrans To\ &Previous\ Error<Tab>[s &Предыдущая\ ошибка<Tab>[s
+menutrans Suggest\ &Corrections<Tab>z= Предложить\ исп&равления<Tab>z=
+menutrans &Repeat\ Correction<Tab>:spellrepall Пов&торить\ исправление\ для\ всех<Tab>spellrepall
+menutrans Set\ Language\ to\ "en" Установить\ язык\ "en"
+menutrans Set\ Language\ to\ "en_au" Установить\ язык\ "en_au"
+menutrans Set\ Language\ to\ "en_ca" Установить\ язык\ "en_ca"
+menutrans Set\ Language\ to\ "en_gb" Установить\ язык\ "en_gb"
+menutrans Set\ Language\ to\ "en_nz" Установить\ язык\ "en_nz"
+menutrans Set\ Language\ to\ "en_us" Установить\ язык\ "en_us"
+menutrans &Find\ More\ Languages &Найти\ больше\ языков
+
+" func! s:SpellLang()
+if !exists("g:menutrans_set_lang_to")
+  let g:menutrans_set_lang_to = 'Установить язык'
+endif
+" }}}
```

Comment:
All these "if not exists" look unneeded and, probably, even harmful. This condition is in use in `$VIMRUNTIME/menu.vim` but what is it doing here? AFAIU, every new menu translation should just overwrite the previous one.

Reply:
> All these "if not exists" look unneeded and, probably, even harmful. This condition is in use in `$VIMRUNTIME/menu.vim` but what is it doing here? AFAIU, every new menu translation should just overwrite the previous one.

Thanks. Yes, you are right.  Although almost all previous translations have retained this, I think it's unresonable. I'll remove it.